common/societies/wc_societies.txt

```diff
@@ -95,10 +94,11 @@ cult_of_the_damned = {
 		}
 		decisions = {
 			the_devil_worshipers_tainted_touch
-			devil_worshipers_dark_healing
+			# devil_worshipers_dark_healing
 			ask_for_dark_gift
 			prisoner_create_monstrous_undead
 		}
+		custom_tooltip = devil_worshipers_dark_healing_power_tooltip
```

Comment:
Maybe show decision, but add text like "Requires Monks and Mystics" if players don't have the DLC?

Reply:
Sounds really bad, imo.
I think that constantly reminding players after game start that they need DLC in order to use that feature it's just a bad manners. Even paradox understands that and just and hide or make inactive content that is DLC-locked.